frontend/components/receipts/transfer/enriched-transfer-view.tsx

```diff
@@ -14,21 +13,18 @@ export const EnrichedTransferView = ({ transaction }: ReceiptComponentProps) =>
     getAssetById: state.getAssetById
   }))
   const assetInfo = getAssetById(transaction.transfer.asset)
-  const { getKeyById } = useWalletStore((state) => ({ getKeyById: state.getKeyById }))
   const { amount } = transaction.transfer
   const decimals = Number(assetInfo?.details?.decimals)
-  const price = amount && decimals ? formatNumber(toBigNum(amount, decimals), decimals) : undefined
+  const formattedAmount = amount && decimals ? formatNumber(toBigNum(amount, decimals), decimals) : undefined
   const symbol = assetInfo?.details?.symbol
-  const keyInfo = getKeyById(transaction.transfer.to)
-  const isOwnKey = keyInfo?.publicKey === transaction.transfer.to
 
   return (
-    <div data-testid={locators.enrichedSection}>
-      <div className="text-xl text-white">
-        {price && symbol ? <PriceWithSymbol price={price} symbol={symbol} /> : null}
+    <BaseTransferView transaction={transaction}>
+      <div data-testid={locators.enrichedSection}>
+        <div className="text-xl text-white">
+          {formattedAmount && symbol ? <PriceWithSymbol price={formattedAmount} symbol={symbol} /> : null}
+        </div>
       </div>
-      <h1 className="text-vega-dark-300 mt-4">To</h1>
-      <VegaKey publicKey={transaction.transfer.to} name={isOwnKey ? `${keyInfo?.name} (own key)` : 'External key'} />
```

Comment:
(blocking): this enriched key work is missing if falling back to the base transfer view implementation

Reply:
this is why the unit tests are failing